### PR TITLE
input bootstrap4, add simple header and footer.

### PR DESCRIPTION
**Pull request by jason-jz-zhu**
1. Finished #1 . 
2. Add simple Header #2 and Footer #3 . Which need to be improved.

### CODE REVIEW

**Review by QiyuLi — CHANGES_REQUESTED** [review]
N/A

File: package.json [hunk 1]
```diff
@@ -42,12 +43,16 @@
     "isomorphic-fetch": "^2.2.1",
     "limax": "^1.3.0",
     "mongoose": "^4.4.20",
-    "react": "^15.1.0",
-    "react-dom": "^15.1.0",
+    "react": "^15.4.2",
+    "react-addons-css-transition-group": "^15.4.2",
+    "react-addons-transition-group": "^15.4.2",
+    "react-bootstrap": "^0.30.7",
```
Comment (QiyuLi) [inline package.json]: Why do we need react-bootstrap?
reactstrap does not need this.

**Review by QiyuLi — CHANGES_REQUESTED** [review]
I think react-bootstrap is not necessary

File: package.json (hunk 1 above)
Comment (jason-jz-zhu) [reply package.json]: Yes. I forgot to remove it. I install it just to test. Thanks.

**Comment by QiyuLi** [comment]
This PR resolve #1 .
See https://help.github.com/articles/closing-issues-via-commit-messages/

File: client/modules/App/components/Header/Header.js [hunk 2]
```diff
@@ -1,45 +1,57 @@
...
+  }
+  toggle() {
+    this.setState({
+      isOpen: !this.state.isOpen,
+    });
+  }
+  render() {
+    return (
+      <div>
+        <Navbar color="faded" fixed="top" light top toggleable>
```
Comment (QiyuLi) [inline client/modules/App/components/Header/Header.js]: Why top here?
The top I'm referring to is the `light top toggleable`

**Review by QiyuLi — CHANGES_REQUESTED** [review]
The top property is not recognized by react

**Review by QiyuLi — APPROVED** [review]

**Pull request merged**